src/util/basketUtils.js

```diff
@@ -19,7 +19,9 @@ function logSetLendAmountError(loanId, err) {
 	}
 }
 
-export function setLendAmount({ amount, apollo, loanId }) {
+export function setLendAmount({
+	amount, apollo, loanId, basketId = '0'
```

Comment:
I don't think you need to pass in this new param at all. You can just pass the int `0` with `id: 0` in the optimisticResponse.

Sorry about my previous comment which was confusing this new param getting passed in with the mutation declaration for $basketId, which is fine as is.

Reply:
Seems like this would show an error when setting the optimisticResponse, but it may not b/c it just needs a placeholder until the real response is returned...

Reply:
Thank you! Setting `id: '0',` within `shop` optimisticResponse fix the error